pkg/ibmvpc/parser.go

```diff
@@ -819,17 +821,18 @@ func getTgwObjects(c *datamodel.ResourcesContainerModel,
 					ResourceType: ResourceTypeTGW,
 					Region:       region,
 				},
-				vpcs:            []*VPC{vpc},
-				availableRoutes: map[string][]*ipblock.IPBlock{},
-				vpcsAPToFilters: map[string][]IPBlockPrefixFilter{},
-				region:          getRegionByName(region, regionToStructMap),
+				vpcs:                []*VPC{vpc},
+				tgwConnList:         tgwConnList,
```

Comment:
Yes. Currently we just copy the list of pointers. We can keep only `relevant` `tgw-connections` - this will require more code than just cloning a slice. Given the low space consumption of a list of pointers I'm not completely sure its worth it.
I don't have a very strong opinion here.  @adisos if you think we should keep only relevant `tgw-connections` I'm fine with it; I'm also fine with the current solution.